src/v3/src/transformer/terminal/transformTerminalTransaction.tsx

```diff
@@ -257,6 +258,14 @@ export const transformTerminalTransaction = (
 
   const formBag: FormBag = createForm();
 
+  // @ts-expect-error Property 'deviceEnrollment' does not exist on type 'IdxContext' ts(2339)
+  const deviceEnrollment = transaction.context?.deviceEnrollment?.value;
+  if (typeof deviceEnrollment !== 'undefined') {
+    if (deviceEnrollment.name === 'oda') {
+      return transformOdaEnrollment({ transaction, formBag, widgetProps });
+    }
+  }
```

Comment:
You may have created the nested if intentionally, if so, just ignore this.
```suggestion
  if (deviceEnrollment?.name === 'oda') {
      return transformOdaEnrollment({ transaction, formBag, widgetProps });
  }
```

Reply:
Oh yeah I left it like this because there is the `'mdm'` enrollment as well that we will handle separately